cmds/shutdown/shutdown.go

```diff
@@ -47,17 +65,44 @@ func main() {
 		op = flag.Args()[0]
 	}
 
-	f, ok := opcodes[op]
+	if op == "" {
+		if flags.h && flags.r {
+			return 0, errors.New("Invalid aruments")
+		} else if flags.h {
+			op = "halt"
+		} else {
+			op = "reboot"
+		}
+
+	} else {
+		if flags.h || flags.r {
+			return 0, errors.New("Invalid aruments")
```

Comment:
Again, just "Invalid arguments" is not enough. In this case, 
"You set an opcode and -h or -r, which is not allowed"

Can we just skip -h and -r as arguments, since we have reboot and halt? Why all these commands when fewer will do?
In u-root we're not required to implement everything.

Reply:
I was deferring to main as it prints out the usage after an error.  I can certainly make the error a bit clearer.

I thought it would be good to implement the standard flags `-k`, `-r`, and `-h` without breaking the existing usage.